Tasks/Gradle/CodeAnalysis/Common/CodeAnalysisResultPublisher.ts

```diff
@@ -44,7 +44,7 @@ export class CodeAnalysisResultPublisher {
                 let reportName = path.basename(resultFile, extension);
 
                 let artifactName = `${prefix}_${reportName}_${analysisResult.toolName}${extension}`;
-                tl.cp('-f', resultFile, path.join(destinationDir, artifactName));
+                tl.cp(resultFile, path.join(destinationDir, artifactName), '-f');
```

Comment:
Do we need to use tl.cp instead of the underlying shell.cp ? Actually copying files means that we can test this (i.e. test that the destination folder contains the test files).